pool/pool_machine.c

```diff
@@ -75,12 +75,12 @@ static struct m0_clink *ready_clink(struct m0_poolmach_state *state)
 	return &state->pst_conf_ready.bc_u.clink;
 }
 
-static bool is_controllerv_or_diskv(const struct m0_conf_obj *obj)
+static bool is_enclosurev_or_diskv(const struct m0_conf_obj *obj)
```

Comment:
Why are these changes required for this patch?

Please add explanation in commit message.

Reply:
This is missed during mapping of node from controller to enclosure.
I have added this detail in the commit message, i will create a separate PR for it. Currently "dgread" branches from both motr and hare are being tested for degraded read after node failure/shutdown cases.

Reply:
Oh, I see the explanation in commit message.
Thank you.

When merge this PR, the explanation should be retained.